doc/sphinx/changes.rst

```diff
@@ -8,6 +8,992 @@ Recent changes
+  noticing when there are two equations of the form ``x = y * q₁ + _`` and
+  ``x = y * q₂ + _`` (or minor variations thereof), suggesting that ``q₁ = q₂``.
+  Users can replace :g:`Z.to_euclidean_division_equations` with
+  :g:`let flags := Z.euclidean_division_equations_flags.default_with Z.euclidean_division_equations_flags.find_duplicate_quotients false in Z.to_euclidean_division_equations_with flags`
+  or, using :g:`Import Z.euclidean_division_equations_flags.`, with
+  :g:`Z.to_euclidean_division_equations_with ltac:(default_with find_duplicate_quotients false)`
+  (`#17934 <https://github.com/coq/coq/pull/17934>`_,
+  by Jason Gross).
+- **Changed:**
+  The opacity/transparency of primitive projections is now attached to the
+  projections themselves, not the compatibility constants, and compatibility
+  constants are always considered transparent
+  (`#18327 <https://github.com/coq/coq/pull/18327>`_,
+  fixes `#18281 <https://github.com/coq/coq/issues/18281>`_,
+  by Jan-Oliver Kaiser and Rodolphe Lepigre).
+- **Changed:**
+  Tactic :g:`intro z` on an existential variable goal forces the resolution
+  of the existential variable into a goal :g:`forall z:?T, ?P`, which
+  becomes :g:`?P` in context :g:`z:?T` after introduction. The
+  existential variable :n:`?P` itself is now defined in a context
+  where the variable of type `?T` is also named :g:`z`, as specified
+  by :tacn:`intro` instead of :g:`x` as it was conventionally the case
+  before
+  (`#18395 <https://github.com/coq/coq/pull/18395>`_,
+  by Hugo Herbelin).
+- **Changed:**
+  syntactic global references passed through the `using` clauses of :tacn:`auto`-like
+  tactics are now handled as plain references rather than interpreted terms. In
+  particular, their typeclass arguments will not be inferred. In general, the previous
+  behaviour can be emulated by replacing `auto using foo` with `pose proof foo; auto`
+  (`#18909 <https://github.com/coq/coq/pull/18909>`_,
+  by Pierre-Marie Pédrot).
+- **Changed:**
+  Use Coqlib's Register mechanism for the generalized rewriting tactic
+  and make the (C)RelationClasses/(C)Morphisms independent of the `rewrite`
+  tactic to ease maintainance.
+  (`#19115 <https://github.com/coq/coq/pull/19115>`_,
+  by Matthieu Sozeau).
+- **Removed:**
+  the clear modifier which was deprecated since 8.17
+  (`#18887 <https://github.com/coq/coq/pull/18887>`_,
+  by Pierre-Marie Pédrot).
+- **Removed:**
+  the `cutrewrite` tactic, which was deprecated since
+  Coq 8.5
+  (`#19027 <https://github.com/coq/coq/pull/19027>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  non-reference hints in `using` clauses of :tacn:`auto`-like tactics
+  (`#19006 <https://github.com/coq/coq/pull/19006>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  the :tacn:`gintuition` tactic, which used to be undocumented
+  until Coq 8.16
+  (`#19129 <https://github.com/coq/coq/pull/19129>`_,
+  by Pierre-Marie Pédrot).
+- **Deprecated:**
+  :tacn:`destauto`.
```

Comment:
```suggestion
  :tacn:`destauto`
```

Reply:
Not so sure since there is a capital on next line, uncapitalized it and replaced dot by a comma